packages/medusa/src/api/middlewares/error-handler.js

```diff
@@ -50,9 +52,12 @@ export default () => {
       case MedusaError.Types.INVALID_ARGUMENT:
         break
       default:
-        errObj.code = "unknown_error"
-        errObj.message = "An unknown error occurred."
-        errObj.type = "unknown_error"
+        // In local environment, we would like the raw error object
+        if (!isLocal) {
+          errObj.code = "unknown_error"
+          errObj.message = "An unknown error occurred."
+          errObj.type = "unknown_error"
+        }
```

Comment:
The issue is, that for integration tests the logs are not always showing. I've found this to be quite annoying, so I usually comment out this part of the error handler, which is not optimal. 

Though, I think you raise a good point, would it make sense to instead create a ticket, that includes a bit of research on how we can improve logging in or at least make sure that it's always displayed when running our test suite?

Reply:
Yes I think that is an excellent idea! Also are you aware of `verbose: true` in `setupServer` this will display anything that is being logged when debugging

Reply:
Yes, I am. Though I've experienced quite a lot the flag having no effect at all, which is why I added this one in the first place.

Reply:
Btw. I am usually using `console.error` to try to provoke the logs in the integration tests, but does not seem to work. Do I need to use a different logger?